### PR TITLE
Refactor file structure

### PR DESCRIPTION
**Pull request by lishaduck**
Also, fixes current_directory/cwd error, and rewrites the readme to use markdown, @infinityJKA. Reformats the code with black, sorts imports with isort, and manages dependencies with poetry. Adds a gitignore. Adds status checks.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
```
Comment (lishaduck) [inline .github/dependabot.yml:11]: Checks for dep updates daily

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,312 @@
+# THESE SHOULD BE IGNORED, BUT ARE NOT YET, DUE TO LACK OF PROPER GENERATION AT STARTUP
+#save.dat
+#autosave.dat
```
Comment (lishaduck) [inline .gitignore:3]:
```suggestion
save.dat
autosave.dat
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| w          | UP              |
+| r          | EQUIP           |
+| t          | PARTY           |
+| a          | LEFT            |
+| s          | DOWN            |
+| d          | RIGHT           |
+| f          | ITEM            |
+| g          | TALK            |
+| arrow keys | movement        |
+| space key  | for TALK        |
```
Comment (lishaduck) [inline README.md:28]: Converted the list to a table

Comment (lishaduck) [inline README.md:28]: Converted the list to a table.

File: bipole5/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+"""Bipole V! A game.
+
+TODO: Ought BipoleV.py to be renamed to __init__.py, and replace this?
```
Comment (lishaduck) [inline bipole5/__init__.py:3]: ?

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+authors = ["InfinityJKA <jkajustin@gmail.com>"]
+description = "#26 in the Bipole Series"
+name = "bipole5"
+readme = "README.md"
+version = "1.0.0"
+
+[tool.poetry.dependencies]
+numpy = "^1.23.5"
+pillow = "^9.3.0"
+python = "^3.10"
```
Comment (lishaduck) [inline pyproject.toml]: Dependencies for the project.

Comment (lishaduck) [inline pyproject.toml]:
```suggestion
numpy = "^1.23.5"
pillow = "^9.3.0"
python = "^3.7"
```

File: pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+authors = ["InfinityJKA <jkajustin@gmail.com>"]
```
Comment (lishaduck) [inline pyproject.toml:2]: Is this the right email?

File: bipole5/BipoleV.py [hunk 7]
```diff
@@ -1,95 +1,165 @@
...
+    from . import characters
+    from . import enemies
+    from . import equipment
+    from . import maps
+
+from pathlib import Path
+
+from PIL import Image, ImageTk
+
+current_directory: Path = Path(os.path.realpath(__file__)).parent
```
Comment (lishaduck) [inline bipole5/BipoleV.py:29]: This is a better way to get the file's location, @infinityJKA.

File: bipole5/BipoleV.py [hunk 8]
```diff
@@ -749,29 +1310,39 @@ def disable_keys():
...
     global dialogue
     global dialogue_index
     dialogue_index = 1
-    file_contents = open(current_directory+"/dialogue/"+maps.current_location[4]+"/"+input_file+".txt")
-    dialogue = [line.rstrip('\n') for line in file_contents]
+    file_contents = open(
+        current_directory
+        / "dialogue"
+        / maps.current_location[4]
+        / (input_file + ".txt"),
```
Comment (lishaduck) [inline bipole5/BipoleV.py:1327]: Use the '/' operator to append strings to a path, @infinityJKA.

File: bipole5/BipoleV.py [hunk 9]
```diff
@@ -749,29 +1310,39 @@ def disable_keys():
...
+        encoding="UTF-8",
+    )
```
Comment (lishaduck) [inline bipole5/BipoleV.py:1345]: Use the '/' operator to append strings to a path, @infinityJKA.
Oh, and when opening in text ("t") mode, always specify an encoding.

**Comment by lishaduck** [comment]
What did you use to build the output folder, @infinityJKA?

**Comment by infinityJKA** [comment]
I think I used auto-py-to-exe

**Comment by lishaduck** [comment]
I think packaging works now, @infinityJKA. The only thing would be uploading the build artifacts to releases. I'd think there's a way to automate it, but I don't see one.

**Comment by lishaduck** [comment]
Packaging should be fixed now, @infinityJKA.

**Comment by lishaduck** [comment]
Should I break this into smaller PRs, 1 at a time, @infinityJKA?

**Comment by infinityJKA** [comment]
> Should I break this into smaller PRs, 1 at a time, @infinityJKA?

Idk, if you want

**Comment by lishaduck** [comment]
I've decided that I'm too busy to update this, and it's useless in this condition, so I'm closing it.

**Comment by infinityJKA** [comment]
real

**Comment by lishaduck** [comment]
I'm going to prune the repo at the very least now that I can run it again. I'll try to be more conservative this time.

**Comment by infinityJKA** [comment]
@lishaduck should i be scared

**Comment by lishaduck** [comment]
No. I'm going 1 commit @ a time

**Comment by infinityJKA** [comment]
Remember to commit responsibly!

**Comment by lishaduck** [comment]
I've double checked, #2 still works fully on a managed mac w/py3.11.6!

**Comment by lishaduck** [comment]
#3 is just docs, but I checked anyway, and it too works.

**Pull request closed without merging**